### ISSUE TITLE
Deprecation warnings on tasks

### ISSUE THREAD

**Issue by omricohen** [mintdigital/asset_hat#17]
Getting the following deprecation warnings, should be an easy fix. I can submit a pull request if you want. Let me know.

```
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/css.rb:5:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/css.rb:21:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/css.rb:43:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/css.rb:68:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/css.rb:158:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/js.rb:5:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/js.rb:34:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks/js.rb:107:in `block (2 levels) in <top (required)>'
WARNING: 'task :t, arg, :needs => [deps]' is deprecated.  Please use 'task :t, [args] => [deps]' instead.
at /home/deploy/shared/bundle/ruby/1.9.1/gems/asset_hat-0.4.1/lib/asset_hat/tasks.rb:7:in `block in <top (required)>'
```

**Comment by rondevera** [comment]
Thanks for reporting this! I've merged the pull request from issue #16 into the `development` branch, and it's being tested this week. A new patch version should be out very soon.

Cheers,
Ron

**Comment by omricohen** [comment]
youre awesome. thanks!
